homeassistant/components/flume/sensor.py

```diff
@@ -24,6 +28,47 @@
 from .coordinator import FlumeDeviceDataUpdateCoordinator
 from .entity import FlumeEntity
 
+FLUME_QUERIES_SENSOR: tuple[SensorEntityDescription, ...] = (
```

Comment:
It is normally recommended to keep `EntityDescription` definitions inside each platform, so I have moved them to `sensor.py` before adding the device class.

Reply:
I was wondering when we should do this. I guess the answer is now :)